packages/app/src/app/graphql/types.ts

```diff
@@ -1571,7 +1571,7 @@ export type TemplateFragmentDashboardFragment = {
 
 export type TeamFragmentDashboardFragment = { __typename?: 'Team' } & Pick<
   Team,
-  'id' | 'name' | 'description' | 'creatorId' | 'avatarUrl'
+  'id' | 'name' | 'description' | 'creatorId' | 'avatarUrl' | 'joinedPilotAt'
```

Comment:
Added `joinedPilotAt` to the fragment because now, we use it in multiple places. This would get replaced by `subscriptions` once we migrate pro users to the new payments stack